newIDE/app/src/Utils/GDevelopServices/Asset.js

```diff
@@ -74,6 +74,8 @@ export type AssetPack = {|
   tag: string,
   thumbnailUrl: string,
   assetsCount: number,
+  externalWebLink?: ?string,
+  userFriendlyPrice?: ?string,
```

Comment:
is this a good idea to handle the price ourselves? This forces us to monitor packs and update if they do.
Should this be more generic like "Platformer - Paid pack"?

Reply:
For now I would say it's important because this can makes the difference between clicking on the asset pack to check it or not. As we'll start with paid asset packs made by community members, we can rely on them, by asking paid pack authors to tell us if they change the price :)